src/index.d.ts

```diff
@@ -1,3 +1,6 @@
+
```

Comment:
should use Standard style like the rest of the repo for consistency -- no newline at the top, no semicolons, 2 space indentation. see my format commit